components/stage/scripts/create.py

```diff
@@ -42,25 +44,30 @@ def install_stage():
     utils.mkdir(stage_home)
     utils.mkdir(stage_log_path)
 
-    utils.create_service_user(stage_user, stage_home)
-
-    ctx.logger.info('Installing NodeJS...')
-    nodejs = utils.download_cloudify_resource(nodejs_source_url,
-                                              STAGE_SERVICE_NAME)
-    utils.untar(nodejs, nodejs_home)
-
     ctx.logger.info('Installing Cloudify Stage (UI)...')
-    stage = utils.download_cloudify_resource(stage_source_url,
-                                             STAGE_SERVICE_NAME)
-    utils.untar(stage, stage_home)
-
-    ctx.logger.info('Fixing permissions...')
-    utils.chown(stage_user, stage_group, stage_home)
-    utils.chown(stage_user, stage_group, nodejs_home)
-    utils.chown(stage_user, stage_group, stage_log_path)
-
-    utils.logrotate(STAGE_SERVICE_NAME)
-    utils.systemd.configure(STAGE_SERVICE_NAME)
+    stage = utils.download_cloudify_resource(
+            stage_source_url, STAGE_SERVICE_NAME, avoid_failure=True)
+    if not stage:
+        ctx.instance.runtime_properties['ignore_ui'] = 'True'
+        print "***ignore ui***"
+    ignore_ui = ctx.instance.runtime_properties['ignore_ui']
+    print ctx.instance.runtime_properties['ignore_ui'].__class__
+    print "ignore_ui={0}".format(ignore_ui)
+    if ignore_ui != 'True':
```

Comment:
Maybe in order not to get all the script indented just do something like:

if skip_stage_installation:
    return